main/managementinterface.go

```diff
@@ -407,6 +407,19 @@ func handleSettingsSetRequest(w http.ResponseWriter, r *http.Request) {
 					case "RadarRange":
 						globalSettings.RadarRange = int(val.(float64))
 						radarUpdate.SendJSON(globalSettings)
+					// Serial Management with Capabilities and Baud support
+					case "SerialOutput":
+						modelItem := val.(map[string]interface{})
+						if(modelItem["DeviceString"]!=nil && len(modelItem["DeviceString"].(string))>0 &&
```

Comment:
Can you please try to follow the coding style here? No (...) around if-conditions if it's not needed, spaces around operators, ...